{"name": "Login", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n\tGetHandleVerifier [0x00007FF74F862142+3514994]\n\t(No symbol) [0x00007FF74F480CE2]\n\t(No symbol) [0x00007FF74F3276AA]\n\t(No symbol) [0x00007FF74F371860]\n\t(No symbol) [0x00007FF74F37197C]\n\t(No symbol) [0x00007FF74F3B4EE7]\n\t(No symbol) [0x00007FF74F39602F]\n\t(No symbol) [0x00007FF74F3B28F6]\n\t(No symbol) [0x00007FF74F395D93]\n\t(No symbol) [0x00007FF74F364BDC]\n\t(No symbol) [0x00007FF74F365C64]\n\tGetHandleVerifier [0x00007FF74F88E16B+3695259]\n\tGetHandleVerifier [0x00007FF74F8E6737+4057191]\n\tGetHandleVerifier [0x00007FF74F8DE4E3+4023827]\n\tGetHandleVerifier [0x00007FF74F5B04F9+689705]\n\t(No symbol) [0x00007FF74F48C048]\n\t(No symbol) [0x00007FF74F488044]\n\t(No symbol) [0x00007FF74F4881C9]\n\t(No symbol) [0x00007FF74F4788C4]\n\tBaseThreadInitThunk [0x00007FFE6E0D257D+29]\n\tRtlUserThreadStart [0x00007FFE6F60AA58+40]\n\n", "trace": "  File \"C:\\Users\\ASUS\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\ASUS\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"src\\features\\steps\\login.py\", line 13, in login_user\n    LoginPage(context.driver).login_user(username, password)\n  File \"C:\\Users\\ASUS\\Documents\\Python\\src\\pages\\login\\login.py\", line 33, in login_user\n    self.username(username)\n  File \"C:\\Users\\ASUS\\Documents\\Python\\src\\pages\\login\\login.py\", line 13, in username\n    user = WebDriverWait(self.driver, 10).until(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ASUS\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Given I open the saucedemo page", "status": "passed", "start": 1704828249368, "stop": 1704828250071}, {"name": "When Enter username \"standard_user\" and password \"secret_sauce\"", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n\tGetHandleVerifier [0x00007FF74F862142+3514994]\n\t(No symbol) [0x00007FF74F480CE2]\n\t(No symbol) [0x00007FF74F3276AA]\n\t(No symbol) [0x00007FF74F371860]\n\t(No symbol) [0x00007FF74F37197C]\n\t(No symbol) [0x00007FF74F3B4EE7]\n\t(No symbol) [0x00007FF74F39602F]\n\t(No symbol) [0x00007FF74F3B28F6]\n\t(No symbol) [0x00007FF74F395D93]\n\t(No symbol) [0x00007FF74F364BDC]\n\t(No symbol) [0x00007FF74F365C64]\n\tGetHandleVerifier [0x00007FF74F88E16B+3695259]\n\tGetHandleVerifier [0x00007FF74F8E6737+4057191]\n\tGetHandleVerifier [0x00007FF74F8DE4E3+4023827]\n\tGetHandleVerifier [0x00007FF74F5B04F9+689705]\n\t(No symbol) [0x00007FF74F48C048]\n\t(No symbol) [0x00007FF74F488044]\n\t(No symbol) [0x00007FF74F4881C9]\n\t(No symbol) [0x00007FF74F4788C4]\n\tBaseThreadInitThunk [0x00007FFE6E0D257D+29]\n\tRtlUserThreadStart [0x00007FFE6F60AA58+40]\n\n", "trace": "  File \"C:\\Users\\ASUS\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\ASUS\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"src\\features\\steps\\login.py\", line 13, in login_user\n    LoginPage(context.driver).login_user(username, password)\n  File \"C:\\Users\\ASUS\\Documents\\Python\\src\\pages\\login\\login.py\", line 33, in login_user\n    self.username(username)\n  File \"C:\\Users\\ASUS\\Documents\\Python\\src\\pages\\login\\login.py\", line 13, in username\n    user = WebDriverWait(self.driver, 10).until(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ASUS\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1704828250071, "stop": 1704828260355}, {"name": "Then Click on login button", "status": "skipped", "start": 1704828282904, "stop": 1704828282904}], "start": 1704828245624, "stop": 1704828282904, "uuid": "f74201ef-fb23-4663-8cf2-7cbaab44cc66", "historyId": "b004641467c49656e31d9a4ca6a7eae3", "fullName": "Perform login: Login", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "LoginOk"}, {"name": "feature", "value": "Perform login"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}